python_modules/libraries/dagster-aws/dagster_aws_tests/s3_tests/test_io_manager.py

```diff
@@ -225,29 +223,3 @@ def test_s3_pickle_io_manager_asset_execution(mock_s3_bucket):
             "/".join(["dagster", "storage", result.run_id, "graph_asset.first_op", "result"]),
         ),
     }
-
-
-def test_nothing(mock_s3_bucket):
```

Comment:
this test and the similar tests for GCS and ADLS2 I/O managers aren't testing any functionality specific to the I/O manager anymore. They're redundant with the tests of the main I/O manager